symfony/client/src/orangehrmTimePlugin/pages/configure/TimeSheetPeriod.vue

```diff
@@ -0,0 +1,134 @@
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-background-container">
+    <div class="orangehrm-card-container">
+      <oxd-text class="orangehrm-main-title">
+        {{ $t('time.timesheet_period_config') }}
+      </oxd-text>
+      <oxd-divider />
+      <oxd-form :loading="isLoading" @submitValid="onSave">
+        <oxd-form-row>
+          <oxd-grid :cols="2" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-field
+                type="select"
+                :label="$t('time.add_timesheet_period_config')"
+                :options="days"
+                :rules="rules.startDay"
+                v-model="timeSheetPeriod.startDay"
+                required
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-divider />
+        <oxd-form-actions>
+          <required-text />
+          <submit-button />
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+</template>
+
+<script>
+import {APIService} from '@ohrm/core/util/services/api.service';
+import {enGB} from 'date-fns/locale';
+import {required} from '@/core/util/validation/rules';
+import {navigate} from '@/core/util/helper/navigation';
+
+const timeSheetPeriodModel = {
+  startDay: null,
+};
+
+export default {
+  data() {
+    return {
+      isLoading: false,
+      timeSheetPeriod: {...timeSheetPeriodModel},
+      rules: {
+        startDay: [required],
+      },
+    };
+  },
+  setup() {
+    const http = new APIService(
+      window.appGlobal.baseUrl,
+      '/api/v2/time/time-sheet-period',
+    );
+    return {
+      http,
+    };
+  },
+  methods: {
+    onSave() {
+      this.isLoading = true;
+      this.http
+        .request({
+          method: 'PUT',
+          data: {
+            startDay: this.timeSheetPeriod.startDay?.id,
+          },
+        })
+        .then(() => {
+          return this.$toast.saveSuccess();
+        })
+        .then(() => {
+          navigate('/time/viewEmployeeTimesheet');
+        });
+    },
+    updateTimeSheetPeriodModel(day) {
+      this.timeSheetPeriod.startDay = this.days.find(d => {
+        return d.id === day;
+      });
+    },
+  },
+  computed: {
+    days() {
+      return Array(7)
+        .fill('')
+        .map((...[, index]) => {
+          return {
+            id: index + 1,
+            label: enGB.localize.day(index, {
```

Comment:
@Super-Chama shall we move this function also into the helper class. I saw this used some other places